README.md

```
@@ -120,7 +120,15 @@ $ icloud devices export "Example iPhone" --output ./iphone.json
 $ icloud calendar events --username jappleseed@apple.com --period week
 $ icloud contacts me --username jappleseed@apple.com
 $ icloud drive list /Documents --username jappleseed@apple.com
+$ icloud photos libraries --username jappleseed@apple.com
```

markdownlint-cli2 (0.22.0)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)